index.html

```diff
@@ -0,0 +1,141 @@
+          <div><img class="person-card__image" src="./images/user-cover-1.png"
+              alt="a female nurse in sky blue uniforms"></div>
+          <div class="person-card__content">
+            <h5 class="person-card__person-name">Julian Jameson</h5>
+            <h6 class="person-card__proffesion">Proffession</h6>
+            <div class="person-card__socials-container">
+              <div><img src="./images/ant-design_facebook-filled.png" alt="Facebook icon"></div>
+              <div><img src="./images/ant-design_instagram-outlined.png" alt="Instagram icon"></div>
+              <div><img src="./images/ant-design_twitter-outlined.png" alt="twitter icon"></div>
+            </div>
+          </div>
+        </div>
+        <div class="person-card">
+          <div><img class="person-card__image" src="./images/user-cover-2.png" alt="a male doctor on a red background">
+          </div>
+          <div class="person-card__content">
+            <h5 class="person-card__person-name">Julian Jameson</h5>
+            <h6 class="person-card__proffesion">Proffession</h6>
+            <div class="person-card__socials-container">
+              <div><img src="./images/ant-design_facebook-filled.png" alt="Facebook icon"></div>
+              <div><img src="./images/ant-design_instagram-outlined.png" alt="Instagram icon"></div>
+              <div><img src="./images/ant-design_twitter-outlined.png" alt="twitter icon"></div>
+            </div>
+          </div>
+        </div>
+        <div class="person-card">
+          <div><img class="person-card__image" src="./images/user-cover-3.png" alt="a female doctor with stethoscope">
+          </div>
+          <div class="person-card__content">
+            <h5 class="person-card__person-name">Julian Jameson</h5>
+            <h6 class="person-card__proffesion">Proffession</h6>
+            <div class="person-card__socials-container">
+              <div><img src="./images/ant-design_facebook-filled.png" alt="Facebook icon"></div>
+              <div><img src="./images/ant-design_instagram-outlined.png" alt="Instagram icon"></div>
+              <div><img src="./images/ant-design_twitter-outlined.png" alt="twitter icon"></div>
+            </div>
+          </div>
+        </div>
+        <div class="person-card">
+          <div><img class="person-card__image" src="./images/user-cover-4.png"
+              alt="a female doctor standing with a stethoscope"></div>
+          <div class="person-card__content">
+            <h5 class="person-card__person-name">Julian Jameson</h5>
+            <h6 class="person-card__proffesion">Proffession</h6>
+            <div class="person-card__socials-container">
+              <div><img src="./images/ant-design_facebook-filled.png" alt="Facebook icon"></div>
+              <div><img src="./images/ant-design_instagram-outlined.png" alt="Instagram icon"></div>
+              <div><img src="./images/ant-design_twitter-outlined.png" alt="twitter icon"></div>
```

Comment:
You don't really need these `div's` but if you want to have them there, you should give them classes. As for the images inside, you should give them a class like `person-card__social-icon` to follow BEM.

Reply:
I usually put an image inside a div when the parent is a flex element to ensure it behaves consistently with the `flex-shrink` property
I will try to give them classes to follow BEM or find a workaround that removes those `divs` with no classes
👍